Assets/Scripts/Player/BasicPlayerShooting.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BasicPlayerShooting : MonoBehaviour
+{
+    public float fireRate;
+    public float speed;
+    public int magSize;
+    public GameObject bulletPrefab;
+    public GameObject bullet;
+    public Transform spawnR;
+    public Transform spawnL;
+    public bool alternate;
+
+    private float timerOne;
+    private float timerTwo;
+
+    private void Start()
+    {
+        timerOne = fireRate;
+    }
+    private void Update()
+    {
+        if (Input.GetKey(KeyCode.Space))
+        {
+            if (alternate)
+            {
+                if (timerOne < 0)
+                {
+                    bullet = Instantiate(bulletPrefab, spawnR.position, spawnR.rotation);
+                    bullet.GetComponent<Rigidbody>().velocity = spawnR.forward * speed;
+                    timerOne = fireRate;
+                    timerTwo = fireRate / 2f;
+                }
+                if (timerTwo < 0)
+                {
+                    bullet = Instantiate(bulletPrefab, spawnL.position, spawnL.rotation);
+                    bullet.GetComponent<Rigidbody>().velocity = spawnL.forward * speed;
+                    timerTwo = fireRate;
+                }
+            }
+            else 
+            {
+                if (timerOne < 0)
+                {
+                    bullet = Instantiate(bulletPrefab, spawnR.position, spawnR.rotation);
+                    bullet.GetComponent<Rigidbody>().velocity = spawnR.forward * speed;
+                    bullet = Instantiate(bulletPrefab, spawnL.position, spawnL.rotation);
+                    bullet.GetComponent<Rigidbody>().velocity = spawnL.forward * speed;
+                    timerOne = fireRate;
+                }
+            }          
+        }
+        timerOne -= Time.deltaTime;
+        timerTwo -= Time.deltaTime;
```

Comment:
I'm wondering if this should be moved to the if statement with alternate (maybe line 42?). Really nit picky but timerTwo shouldn't be decreasing if it's not being used for super minor perf reasons. Also I feel like a possible issue that could come up is that you are decreasing timerTwo when it was never technically initialized